.github/workflows/app-anonymous-users-csharp.yml

```diff
@@ -0,0 +1,70 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and deploy ASP.Net Core app (App-Anonymous-Users) to Azure Web App
```

Comment:
Please run build and deployment pipeline and share the screenshots.

Reply:
![image](https://user-images.githubusercontent.com/117716554/217791829-a6e95a12-fc89-4e98-8272-189e4e548dab.png)